### PR TITLE
chore: Updated Dev Documentation In DEV.md And Node Engine Minimum Version

### PR DESCRIPTION
**Pull request by kelvinator07**
This PR fixes 2 things
1 - A typo in the dev documentation for verifying the docker compose version 
2 - The node engine in package.json to allow to run the project on node version greater than or equal to v14. (Base Version Specified On The Dev Documentation)

### CODE REVIEW

File: DEV.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ $ jq --version
 jq-1.6
 $ docker --version
 Docker version 20.10.7, build f0df350
-$ docker compose version
+$ docker-compose --version
```
Comment (krtk6160) [inline DEV.md:17]: The original command works fine for me:
image

Comment (kelvinator07) [reply DEV.md:17]: Doesn't Work For Me.. Thats Strange
Screenshot 2022-01-17 at 10 40 34

**Comment by bodymindarts** [comment]
I don't think we should relax the node requirements. 14 is no longer under LTS. Instead we should tighten the dev requirements. https://github.com/GaloyMoney/galoy/pull/967

File: DEV.md (hunk 1 above)
Comment (krtk6160) [reply DEV.md:17]: Are you running docker-compose v2?

Comment (vindard) [reply DEV.md:17]: Needs docker-compose v2, this was a deliberate change we made some months back when we upgraded docker-compose requirements from v1 to v2 (see PR #779)

**Comment by kelvinator07** [comment]
> I don't think we should relax the node requirements. 14 will not be LTS much longer. Instead we should tighten the dev requirements. #967

Sounds Better, You Might Also Want To Update The Node Engine in Package.json to >=16

File: DEV.md (hunk 1 above)
Comment (kelvinator07) [reply DEV.md:17]: I'm Still On v1, Will Upgrade To v2

**Comment by kelvinator07** [comment]
I Will Be Closing This PR Since All Issues Has Been Resolved. Thanks

**Pull request closed without merging**